test/48_air_mmult_2x2_channel/test.cpp

```diff
@@ -0,0 +1,154 @@
+//===- test.cpp -------------------------------------------------*- C++ -*-===//
+//
+// Copyright (C) 2023, Advanced Micro Devices, Inc.
+// SPDX-License-Identifier: MIT
+//
+//===----------------------------------------------------------------------===//
+
+#include <cassert>
+#include <cstdio>
+#include <dlfcn.h>
+#include <fcntl.h>
+#include <iostream>
+#include <stdlib.h>
+#include <sys/mman.h>
+#include <thread>
+#include <unistd.h>
+#include <vector>
+
+#include "air.hpp"
+#include "test_library.h"
+
+#define VERBOSE 0
+#define PROFILE 0
+
+namespace {
+
+template <typename T>
+void mm_out(tensor_t<T, 2> *a, tensor_t<T, 2> *b, tensor_t<T, 2> *r) {
+  size_t a_h = a->shape[0];
+  size_t a_w = a->shape[1];
+  size_t b_h = b->shape[0];
+  size_t b_w = b->shape[1];
+  assert(a_w == b_h);
+
+  for (size_t i = 0; i < a_h; i++) {
+    for (size_t j = 0; j < b_w; j++) {
+      size_t idx = i * b_w + j;
+      r->data[idx] = (T)(0);
+      for (size_t k = 0, ke = a_w; k < a_w; k++) {
+        T _a = a->data[i * a_w + k];
+        T _b = b->data[k * b_w + j];
+        r->data[idx] += _a * _b;
+      }
+    }
+  }
+}
+
+} // namespace
+
+int main(int argc, char *argv[]) {
+  uint64_t col = 5;
+  uint64_t row = 3;
+
```

Comment:
Need to call `air_init` here, otherwise wouldn't work for me on vck5000.
```C++
  hsa_status_t init_status = air_init();
  if (init_status != HSA_STATUS_SUCCESS) {
    std::cout << "air_init() failed. Exiting" << std::endl;
    return -1;
  }
```

Reply:
Thanks for looking at this PR. Call to air_init is added now.